graphix/gflow.py

```diff
@@ -19,7 +19,7 @@
 from graphix.linalg import MatGF2
 
 
-def gflow(graph, input, output, meas_planes, mode="single"):
+def gflow(graph, input, output, meas_planes, mode="single", pattern=None):
```

Comment:
I don't think it's a good idea to have pattern as optional argument in `gflow`. having `gflow_from_pattern` is enough. can we get rid of added codes to `gflow`?

Reply:
This was difinitely unnecessary. I deleted it. Thank you.